resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -73,13 +71,10 @@
 			"r_c asks m_c to speak up while chatting together.",
 			"m_c overthinks something {PRONOUN/m_c/subject} said to r_c.",
 			"m_c does something embarrassing in front of r_c, but r_c doesn't seem to mind.",
-			"m_c apologizes to r_c, but r_c doesn't know why.",

```

Comment:
Removed

Reply:
Moved this out of the "nervous" interactions to the general neutral interactions!